x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -1,5 +1,432 @@
 [[update-lifecycle-policy]]
 == Update lifecycle policy
 
-Discuss how one updates an existing policy (when allowed, when not),
- as well as all the stuff about switching policies on indices.
+Updating existing ILM policies is useful to fix mistakes or change
+strategies for newly created indices. Existing indices that are being
+managed by a policy that changes will be modified as well. So that
```

Comment:
This sounds a bit ambiguous, it sounds like the existing indices will be modified at the same time as updating the policy for new indices

Reply:
you're right. this is super weird :)